addons/big-save-button/highlight.css

```diff
@@ -0,0 +1,3 @@
+[class*="menu-bar_menu-bar-item"]:hover {
```

Comment:
> Ideally, the button should only be highlighted if it is clickable. Currently, the background is shown even when there is no text content, and probably while saving too.

I wish we could use this:
```css
[class*="menu-bar_menu-bar-item"]:has([class*="save-status_save-now"] span):hover
```
but [`:has()` needs to be enabled through a flag in Firefox](https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility).

Reply:
Anyone have any ideas of how we could make that work across browsers? Or if maybe we should disable that option in Firefox or something?

Reply:
> Or if maybe we should disable that option in Firefox or something?

I don't want to do that considering this is going to be enabled by default, and also since W_L suggested making this enabled by default (which makes sense).

Reply:
Wait, could we just select `[class*="save-status_save-now"] span` instead? It looks identical to me that way.

Oh, but with Compact editor on, the span element dips down a little, so there is actually a difference by selecting the spam element. This might actually be worth tackling because it means the clickable area is being extended beyond the menu bar. Would you say that's intentional?

Reply:
> Wait, could we just select `[class*="save-status_save-now"] span` instead? It looks identical to me that way.
> 
> Oh, but with Compact editor on, the span element dips down a little, so there is actually a difference by selecting the spam element. This might actually be worth tackling because it means the clickable area is being extended beyond the menu bar. Would you say that's intentional?

I'm thinking that's a bug with the compact editor? I'll need to check it out first. It can probably just be fixed, though.

Reply:
Can't `[class*="save-status_save-now"] span:hover` be selected directly?

Reply:
`[class*="save-status_save-now"]:hover` seems to work fine with my styles below.